addons/im_livechat/static/src/legacy/public_livechat_chatbot.js

```diff
@@ -168,9 +168,9 @@ const QWeb = core.qweb;
         });
 
         const welcomeMessagesIds = welcomeMessages.map(welcomeMessage => welcomeMessage.id);
-        this.messaging.publicLivechatGlobal.livechatButtonView.update({
+        this.messaging.publicLivechatGlobal.update({
             messages: replace(
-                this.messaging.publicLivechatGlobal.livechatButtonView.messages.filter((message) => {
+                this.messaging.publicLivechatGlobal.messages.filter((message) => {
```

Comment:
perhaps now is the good time to remove this useless filter?

Reply:
I'll make a dedicated PR just for this `filter`, to make sure it doesn't break runbot, and isolate the commit in case it introduces issues in the code and we need to revert/change the behaviour of the code in consequence